store/types/pruning.go

```diff
@@ -1,66 +1,77 @@
 package types
 
+import "fmt"
+
+// Pruning option string constants
+const (
+	PruningOptionDefault    = "default"
+	PruningOptionEverything = "everything"
+	PruningOptionNothing    = "nothing"
+	PruningOptionCustom     = "custom"
+)
+
 var (
-	// PruneEverything defines a pruning strategy where all committed states will
-	// be deleted, persisting only the current state.
-	PruneEverything = PruningOptions{
-		KeepEvery:     1,
-		SnapshotEvery: 0,
-	}
+	// PruneDefault defines a pruning strategy where the last 100 heights are kept
+	// in addition to every 500th and where to-be pruned heights are pruned at
+	// every 10th height.
+	PruneDefault = NewPruningOptions(100, 500, 10)
```

Comment:
Note that this persists significantly more data than the old pruning. With 0.38.4, it only kept every 10000 height (plus the last height divisible by 100), while this keeps every 500 heights; so 20 times more.

Reply:
Correct.